### PR TITLE
FRO-70: ZURI CHAT-Fix the admin settings home page and route properly.

### PR DESCRIPTION
**Pull request by Chisquare7**
Linear Ticket link: https://linear.app/team-brainbox/issue/FRO-70/zuri-chat-fix-the-admin-settings-home-page-and-route-properly

# Fixes Issue/Linear Ticket
**My PR Fixes ID-linear_ticket_number_FRO-70**

# Changes proposed
Worked on fixing the admin settings Home page link and route properly

## What were you told to do?
ZURI CHAT-Fix the admin settings home page and route properly.

## What did you do?
- I moved SettingsHome files from the old src folder to new src folder in the protected folder
- I fixed all the import errors and paths of other files associated with settingshome such as adminHeader, adminSidebar
- I fixed the routing links for SettingsHome file in the App.jsx file in the new src folder
- I moved necessary and equivalent assets from old src folder to new src folder

# Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos

Before changes:
image

After changes:
image

### CODE REVIEW

File: packages/main/src/components/admin-header/AdminHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from "react";
+
+import styles from "../admin-header/AdminHead.module.css";
+import { Link } from "react-router-dom";
+
+// icons
```
Comment (ickynavigator) [inline packages/main/src/components/admin-header/AdminHeader.jsx:6]: remove the comments

File: packages/main/src/components/admin-header/AdminHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const currentWorkspace = getCurrentWorkspace();
+  const [workspaceData, setWorkspaceData] = React.useState({});
+
+  useEffect(() => {
+    if (currentWorkspace) {
+      authAxios
+        .get(`/organizations/${currentWorkspace}`)
+        .then(res => {
+          setWorkspaceData(res.data.data);
+          // console.log(res.data.data)
```
Comment (ickynavigator) [inline packages/main/src/components/admin-header/AdminHeader.jsx:27]: remove comment

File: packages/main/src/components/admin-header/AdminHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <div className={styles.adminHeader}>
+      <Link style={{}} to="/admin/settings/home">
+        <div className={styles.organizationLogo}>
+          <FiHome className={styles.icons} />
+          {workspaceData.name}
+        </div>
+      </Link>
+      {/* <div className={styles.menu}>
+        <Link className={styles.menuLink} to="/">
```
Comment (ickynavigator) [inline packages/main/src/components/admin-header/AdminHeader.jsx:44]: remove comment

File: packages/main/src/components/admin-header/utils/Common.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        // console.log(res.data.data)
+        return res.data.data;
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  }
+};
+
+export const ValidateCard = (name, style) => {
```
Comment (ickynavigator) [inline packages/main/src/components/admin-header/utils/Common.jsx:43]: we are removing pricing from the app

why are we requesting cards?

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,311 @@
+import React, { useEffect } from "react";
+import styles from "../admin-sidebar/AdminSidebar.module.css";
+import { Link, NavLink } from "react-router-dom";
+
+// image
```
Comment (ickynavigator) [inline packages/main/src/components/admin-sidebar/AdminSidebar.jsx:5]: comments aren't necessary

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+            <FiCheckSquare className={styles.icon} />
+            About this workspace
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/"
+          >
+            <FiAlertCircle className={styles.icon} />
+            Support for Transport layer security
```
Comment (ickynavigator) [inline packages/main/src/components/admin-sidebar/AdminSidebar.jsx:236]: some of these links aren't necessary

**Review by ickynavigator — CHANGES_REQUESTED** [review]
some of the svgs used can probably be found in the react icons library

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx (hunk 6 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-sidebar/AdminSidebar.jsx:236]: Restructuring the page by removing irrelevant links is on another linear tickets....we are trying to reduce lot of changes per PR

File: packages/main/src/components/admin-header/AdminHeader.jsx (hunk 1 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-header/AdminHeader.jsx]: Done

File: packages/main/src/components/admin-header/AdminHeader.jsx (hunk 2 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-header/AdminHeader.jsx]: Done

File: packages/main/src/components/admin-header/AdminHeader.jsx (hunk 3 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-header/AdminHeader.jsx]: Done

File: packages/main/src/components/admin-header/utils/Common.jsx (hunk 4 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-header/utils/Common.jsx]: Done

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx (hunk 5 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-sidebar/AdminSidebar.jsx]: Done

**Comment by Chisquare7** [comment]
I've made all changes except one because the changes requested is another linear ticket (just trying to reduce changes made per PR)

**Pull request merged**